auth/auth.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require("jsonwebtoken");
+
+module.exports = {
+    createToken : ( user ) => jwt.sign(user, "secretKey", { expiresIn: "1h" }),
+    verifyToken : ( token ) => jwt.verify(token, "secretKey"),
+    bearerToken : (authorization) => {
+        try {
+            const token = authorization.split(" ")[1]
+            return token;
+        } catch (error) {
+            return error;
```

Comment:
Mirá que estas retornando aca un error y es un formato distinto al token en si, o sea.. siguiendo esta logica, si falla y devuelve esto, quien llamó a ésta funcion va a tomar este error como algo valido

Reply:
Lo estuve probando para ver si encuentro el error que me indicás, pero me funciona todo correctamente. El parámetro que debe recibir es una cadena del tipo `"BEARER " + token`, extrae el token y lo devuelve, sino retorna `undefined`, y en el caso de que, por ejemplo, no se le ingresara un parametro a la función, retorna el error `Type error: cannot read properties of undefined (reading 'split')`

Reply:
Me refiero a que la función que llame a bearer, espera recibir un token válido. Si acá estás retornando un error, cómo sabe la función que llamó a ésta que es un error?

Reply:
La función que debería recibir a bearer es `verifyToken`, si esta no recibe un token, o recibe un token expirado, va a retornar algún error describiendo la falla. Yo lo pensé de esa manera.